utils/test/i18n/i18n.util.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { ReactNode, ReactElement } from 'react';
+import { isElement } from 'react-dom/test-utils';
+
+const stringify: (node: ReactNode) => string = (node) => {
+  if (node === undefined || node === null) {
+    return '';
+  }
+
+  if ((node as ReactElement).props) {
+    const props = Object.entries((node as ReactElement).props)
+      .filter(([key]) => key !== 'children')
+      .reduce(
+        (currentProps, [prop, value]) =>
+          `${currentProps} ${prop}=${JSON.stringify(value)}`,
+        ''
+      );
+
+    return `"<${(node as ReactElement).type} ${props.trim()}/>"`;
+  }
+
+  return node.toString();
+};
+
+export const translate: (
+  key: string,
+  inserts?: Record<string, string>
+) => string = (key, insertsObj) => {
+  let inserts = '';
+
+  if (insertsObj) {
+    inserts = JSON.stringify(insertsObj);
+  }
+
+  return `key:"${key}"${inserts ? ` inserts:${inserts}` : ''}`;
+};
+
+export const translateWithFormatting: (
+  key: string,
+  children:
+    | Record<string, string>
+    | ReactNode
+    | [Record<string, string> | ReactNode]
```

Comment:
This seems iffy to me, that's a lot of types

Reply:
I'll play around and see if `Record<string,string>` can be satisfied by `ReactNode`